lib/jenkins_pipeline_builder/extensions/publishers.rb

```diff
@@ -682,3 +676,21 @@
     end
   end
 end
+
+publisher do
+  name :github_pr_coverage_status_reporter
+  plugin_id 'github-pr-coverage-status'
+  description 'Code coverage icon for GitHub pull requests'
+  jenkins_name 'GitHub Pull Request Coverage Status'
+  announced false
+
+  xml do |action|
+    if action == 'compare'
+      send('com.github.terma.jenkins.githubprcoveragestatus.CompareCoverageAction')
```

Comment:
Is this it?  Just one XML tag needed?

Reply:
Yep, there is only 1 XML tag needed, as the configuration is at the global level